website/views/shared/header/header.jade

```diff
@@ -31,9 +31,10 @@
           | {{Math.floor(user.stats.mp)}} / {{user._statsComputed.maxMP}}
     // party
     .party(ng-controller='PartyCtrl')
-      button.party-invite.btn.btn-primary(ng-click="inviteOrStartParty(group)",
-        ng-if="(!party.members || party.memberCount === 1) && user.preferences.displayInviteToPartyWhenPartyIs1",
-        popover="{{!party.members ? env.t('startAParty') : env.t('addToParty')}}", popover-placement="left", popover-trigger="mouseenter")
-        span=env.t("battleWithFriends")
       .herobox-wrap(ng-repeat='profile in partyMinusSelf')
         +herobox()
+      .party-invite-wrap
+        button.party-invite.btn.btn-primary(ng-click="inviteOrStartParty(group)",
+          ng-if="(!party.members || party.memberCount < 4 ) && user.preferences.displayInviteToPartyWhenPartyIs1",
```

Comment:
Whelp, this preference name is now inaccurate. Not sure there's much to do about that unless we want to rename it and do a migration.